src/core/frontend/emu_window.h

```diff
@@ -111,6 +111,16 @@ class EmuWindow : public GraphicsContext {
         return nullptr;
     }
 
+    /**
+     * Save current GraphicsContext.
+     */
+    virtual void SaveContext() {};
+
+    /**
+     * Restore saved GraphicsContext.
+     */
+    virtual void RestoreContext() {};
+
```

Comment:
Appease clang-format
```suggestion
    /**
     * Save current GraphicsContext.
     */
    virtual void SaveContext(){};

    /**
     * Restore saved GraphicsContext.
     */
    virtual void RestoreContext(){};

```